Formula/m/md5deep.rb

```diff
@@ -32,12 +32,19 @@ class Md5deep < Formula
 
   # Fix compilation error due to pointer comparison
   patch do
-    on_sierra :or_newer do
+    on_system :linux, macos: :sierra_or_newer do
       url "https://github.com/jessek/hashdeep/commit/8776134.patch?full_index=1"
       sha256 "3d4e3114aee5505d1336158b76652587fd6f76e1d3af784912277a1f93518c64"
     end
```

Comment:
```suggestion
    url "https://github.com/jessek/hashdeep/commit/8776134.patch?full_index=1"
    sha256 "3d4e3114aee5505d1336158b76652587fd6f76e1d3af784912277a1f93518c64"
```

Let's do this unconditionally. Excluding El Capitan doesn't make any sense.

Reply:
that is true, works for me.